src/Service.Host/client/src/components/reports/SpendBySupplierByDateRange.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useMemo } from 'react';
+import {
+    Loading,
+    ReportTable,
+    BackButton,
+    ExportButton,
+    reportSelectorHelpers
+} from '@linn-it/linn-form-components-library';
+import Grid from '@mui/material/Grid';
+import { useSelector, useDispatch } from 'react-redux';
+import queryString from 'query-string';
+import history from '../../history';
+import config from '../../config';
+import spendBySupplierByDateRangeActions from '../../actions/spendBySupplierByDateRangeActions';
+
+function SpendBySupplierByDateRangeReport() {
+    const options = useMemo(() => queryString.parse(window.location.search) || {}, []);
```

Comment:
useLocation() is clearer/less code

Reply:
+1 from me. would like to eliminate all the window.location.search from all our react code

Reply:
Made the changes. Will remove window.location.search the morning I'm back, test it and we can merge this and check it off the Purchasing Project.

Reply:
Scratch that, just fixed it before I go. All working now. Will resolve and close the other ticket. Just needs added to the menu.